conans/client/conan_api.py

```diff
@@ -929,6 +930,29 @@ def get_default_remote(self):
     def get_remote_by_name(self, remote_name):
         return self._cache.registry.remotes.get(remote_name)
 
+    @api_method
+    def get_recipe_revisions(self, reference, remote_name=None):
+        ref = ConanFileReference.loads(str(reference))
+        if not remote_name:
```

Comment:
Question: Should this "remote or local" logic be in the proxy or not? I see that the `get_file` is similar and it's done in the api.

Reply:
Where is the `get_file` you are referring to? I think that code inside the `ConanProxy` should be related only to remotes, but maybe having a look at the `get_file` I change my mind...

Reply:
Sorry, I meant `get_path`. The `ConanProxy` was designed initially to redirect the calls between the cache and the remotes. But I'm not sure anymore if the proxy changed its roll or if it should disappear. @memsharded

Reply:
Proxy is intended for implementing the caching mechanism: you request the proxy a recipe, it will check in the cache, if not there, download the recipe, then return the path to the recipe.

In this case, nothing is going to be downloaded and cached, so it is good as-is, IMO.